### ISSUE TITLE
eigvalsh documentation error

### ISSUE THREAD

**Issue by proteneer** [tensorflow/tensorflow#24818]
<em>Please make sure that this is a documentation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:doc_template</em>

**System information**
- TensorFlow version: 1.12
- Doc Link:

https://www.tensorflow.org/api_docs/python/tf/linalg/eigvalsh

**Describe the documentation issue**

Note: If your program backpropagates through this function, you should replace it with a call to tf.linalg.eigvalsh (possibly ignoring the second output) to avoid computing the eigen decomposition twice. This is because the eigenvectors are used to compute the gradient w.r.t. the eigenvalues. See _SelfAdjointEigV2Grad in linalg_grad.py.

This warning is quite ambiguous since
1) ling.eigvalsh returns only one output, while linalg.eigh returns two outputs
2) it circularly references itself

**We welcome contributions by users. Will you be able to update submit a PR (use the [doc style guide](https://www.tensorflow.org/community/documentation)) to fix the doc Issue?**

Yes if it devs agree that there's an issue.

**Comment by lamberta** [comment]
Hi Yutong,

That docstring is here: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/ops/linalg_ops.py#L337

Would you be able to make the contribution, please?
